message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -340,6 +340,16 @@
         <param index="6" label="Longitude">Longitude</param>
         <param index="7" label="Altitude" units="m">Altitude, not used. Should be sent as NaN. May be supported in a future version of this message.</param>
       </entry>
+      <entry value="43004" name="MAV_CMD_SET_HAGL" hasLocation="false" isDestination="false">
```

Comment:
Needs to be 43005 to not conflict with the wind command that we haven't brought back from upstream (but should).

Reply:
https://github.com/ArduPilot/mavlink/pull/363

Reply:
changed, and comment on 43004 added